### PR TITLE
feat(temporal): initial support

### PR DESCRIPTION
**Pull request by kosmoz**
Requires: 
- https://github.com/temporalio/helm-charts/pull/555
- https://github.com/glasskube/packages/pull/393
- https://github.com/glasskube/glasskube/pull/1218

### CODE REVIEW

**Comment by pmig** [comment]
@kosmoz I talked with @robholland about the upstream pr. He is eager to merge it, but as he is traveling it will take at least one more week. In the meantime we should either deploy the helm chart to the glasskube charts repo or build the chart and push it to an oci registry so we are not blocked by the upstream.

**Comment by kosmoz** [comment]
@pmig OK thanks, a version of the chart that includes the TTL and default namespace features is now available on charts.glasskube.eu

@christophenne can you check if the package works for you use case?

File: packages/temporal/v1.25.0+1/package.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+name: temporal
+scope: Namespaced
```
Comment (pmig) [inline packages/temporal/v1.25.0+1/package.yaml:3]:
```suggestion
scope: Namespaced
iconUrl: https://avatars.githubusercontent.com/u/56493103
```

Comment (pmig) [reply packages/temporal/v1.25.0+1/package.yaml:3]: NIT: but an icon makes it so much better :-)

**Review by pmig — CHANGES_REQUESTED** [review]

**Review by pmig — DISMISSED** [review]

**Pull request merged**